cmd/block-generator/runner.go

```diff
@@ -0,0 +1,25 @@
+package main
```

Comment:
let's not include this file

Reply:
Oops, this wasn't supposed to be included yet.